src/singularity.ts

```diff
@@ -9,8 +9,8 @@ import { toOrdinal } from './Utility'
 export const updateSingularityPenalties = (): void => {
     const singularityCount = player.singularityCount;
     const color = player.runelevels[6] > 0 ? 'green' : 'red';
-    const platonic = (singularityCount > 36) ? `Platonic Upgrade costs are multiplied by ${format(calculateSingularityDebuff('Platonic Costs', singularityCount), 2, true)}.` : '';
-    const hepteract = (singularityCount > 50) ? `Hepteract Forge costs are multiplied by ${format(calculateSingularityDebuff('Hepteract Costs', singularityCount), 2, true)}.` : '';
+    const platonic = (singularityCount > 36) ? `Platonic Upgrade costs are multiplied by ${format(calculateSingularityDebuff('Platonic Costs', singularityCount), 2, true)}.` : '<span style="color: grey">???????? ??????? ????? ??? ?????????? ?? ???</span> <span style="color: red">(Sing 37)</span>';
+    const hepteract = (singularityCount > 50) ? `Hepteract Forge costs are multiplied by ${format(calculateSingularityDebuff('Hepteract Costs', singularityCount), 2, true)}.` : '<span style="color: grey">????????? ????? ????? ??? ?????????? ?? ???</span> <span style="color: red">(Sing 51)</span>';
```

Comment:
what's the purpose of having these elements?

Reply:
As stated in the original commit, it's used to indicate at what singularity those two penalties unlock as shown in the example image I provided! Prior to this it was just being displayed as empty space if you did not have a high enough singularity for them to be activated, so I simply repurposed that for this. 
![Sing-Example](https://user-images.githubusercontent.com/32188067/193463864-a47db385-e2dd-4663-b73a-7ae294cfd286.PNG)
I just went with the Question Marks as the baseline, we can change it to something like "A new penalty will unlock in Singularity X" if Plat wants. :)